docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -33,6 +33,8 @@ For a video-based overview of the Package Source Mapping feature, consider watch
 To opt into this feature, you must have a `nuget.config` file. Having a single `nuget.config` at the root of your repository is considered a best practice. See [nuget.config documentation](../reference/nuget-config-file.md) to learn more.
 
 Declare your desired package sources in your `nuget.config` file. Following your source declarations, add a `<packageSourceMapping>` element that specifies the desired mappings for each source.

```

Comment:
What if we made these bullet items? I think it would be easier and more readable now that there are a few steps here.

Reply:
Good idea. 

Let me go through it.

Reply:
The bullet items do look better. 

Pushed the change.